### PR TITLE
Dockerfile draft

### PR DESCRIPTION
**Pull request by yookoala**
Please merge this repository AFTER you merged #1 

(Or if there is problem, I'll rebase the branch and fix thing before merge)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Version v0.1.0-dev
+
+FROM ubuntu:16.04
```
Comment (tonyhhyip) [inline Dockerfile]: How about using [node:6](https://hub.docker.com/_/node/) ?

**Comment by yookoala** [comment]
Using nvm might allow us to inject node version through env in the future
(without the need to modify Dockerfile) .

2016年12月7日 21:28，"Tony Yip" <notifications@github.com>寫道：

**Comment by yookoala** [comment]
Added alternative node version support. Also added documentation to README.md

**Comment by yookoala** [comment]
Updated Dockerfile to use node:6 as based image now.

**Comment by yookoala** [comment]
Also optimised based on [Best Practice documentation](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/)

**Comment by yookoala** [comment]
Added multi node-version test on Travis. (v0.10 for docker, also v6 and v7)

**Comment by tonyhhyip** [comment]
Merged into the branch docker

**Pull request closed without merging**